src/applications/check-in/components/ActionItemDisplay.jsx

```diff
@@ -1,36 +1,32 @@
 import React, { useMemo } from 'react';
 import { useSelector } from 'react-redux';
-import { useTranslation } from 'react-i18next';
 import PropTypes from 'prop-types';
 // eslint-disable-next-line import/no-unresolved
 import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports';
 import { createAnalyticsSlug } from '../utils/analytics';
 import { useFormRouting } from '../hooks/useFormRouting';
+
 import { makeSelectApp, makeSelectVeteranData } from '../selectors';
 import { APP_NAMES } from '../utils/appConstants';
 import {
-  hasPhoneAppointments,
   preCheckinAlreadyCompleted,
   getAppointmentId,
 } from '../utils/appointment';
 
 import WhatToDoNext from './WhatToDoNext';
+import PreCheckInSuccessAlert from './PreCheckInSuccessAlert';
 
 const ActionItemDisplay = props => {
   const { router } = props;
   const selectApp = useMemo(makeSelectApp, []);
   const { app } = useSelector(selectApp);
-  const { goToNextPage, jumpToPage } = useFormRouting(router);
+  const { goToNextPage, jumpToPage, pages } = useFormRouting(router);
   const selectVeteranData = useMemo(makeSelectVeteranData, []);
   const { appointments } = useSelector(selectVeteranData);
-  const { t } = useTranslation();
 
   const displaySuccessAlert =
-    app === APP_NAMES.PRE_CHECK_IN && preCheckinAlreadyCompleted(appointments);
-
-  const successMessage = hasPhoneAppointments(appointments)
-    ? t('your-provider-will-call-you-at-your-appointment-time')
-    : t('you-can-check-in-with-your-smartphone');
+    app === APP_NAMES.PRE_CHECK_IN &&
+    (preCheckinAlreadyCompleted(appointments) || pages.length < 5);
```

Comment:
its unlikely but possible for any one of the demos needing update on it own. You may want to write a little util that checks to make sure non of the demo pages are in the form to determine this.

Reply:
We can spin this off in another ticket